### ISSUE TITLE
nodejs package has Optional Chaining and Nullish Coalescing

### ISSUE THREAD

**Issue by jhsu** [lithdew/flatend#4]
When trying to use the nodejs example with node version 12.18.1:

```
/flattend/hello/node_modules/flatend/dist/flatend.js:60
        return this.id?.addr ?? "<anon>";
                       ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:1054:16)
    at Module._compile (internal/modules/cjs/loader.js:1102:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/home/pi/flattend/hello/node_modules/flatend/dist/index.js:4:17)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
```

**Comment by lithdew** [comment]
Ah, the transpiler didn't seem to transpile away the optional chaining/nullish coalescing... might you have any ideas on setting up and picking a preferred transpiler for the NodeJS SDK (not fully up-to-date on the best solutions right now for transpiling NodeJS code)?

Otherwise, I can push a commit to quicky remove the optional chaining/nullish coalescing too.

**Comment by transmissions11** [comment]
@lithdew I think this can be fixed by changing the `target` in tsconfig I believe? You have it set to a version that supports nullish coalescing so it will not attempt to transpile it away.

Setting it to something like ES2017 should fix this issue.

**Comment by lithdew** [comment]
> @lithdew I think this can be fixed by changing the `target` in tsconfig I believe? You have it set to a version that supports nullish coalescing so it will not attempt to transpile it away.
> 
> Setting it to something like ES2017 should fix this issue.

Thanks! Downgraded target to `ES2015` and also removed any uses of bigint literals since TypeScript doesn't support transpiling them. Checking the generated code, I saw places where nullish coalescing and optional chaining got compiled away.

I moved Flatend to v0.0.5.

https://github.com/lithdew/flatend/commit/52fec6161f82003eff4fc9f0bbc9a2595b356a19
https://github.com/lithdew/flatend/commit/feec97f5d26bcea3251773e42b9d0a783a8137d1

Let me know if it works for you @jhsu.
